{"nbformat":4,"nbformat_minor":0,"metadata":{"colab":{"provenance":[],"gpuType":"T4","mount_file_id":"18Fpb_yQAyrz8T2Ji73VdkaFmH4aRQD3m","authorship_tag":"ABX9TyNF+BeO7psRCNzlbF1zXiu3"},"kernelspec":{"name":"python3","display_name":"Python 3"},"language_info":{"name":"python"},"accelerator":"GPU"},"cells":[{"cell_type":"code","execution_count":1,"metadata":{"cellView":"form","id":"STfZQi_nrxES","colab":{"base_uri":"https://localhost:8080/","height":86},"executionInfo":{"status":"ok","timestamp":1709887114387,"user_tz":-60,"elapsed":49656,"user":{"displayName":"Baptiste Roelens","userId":"10230221551692918814"}},"outputId":"30945fca-75ba-4ef8-c3a8-71db008866fc"},"outputs":[{"output_type":"stream","name":"stderr","text":["WARNING:paperscraper.load_dumps: No dump found for biorxiv. Skipping entry.\n","WARNING:paperscraper.load_dumps: No dump found for chemrxiv. Skipping entry.\n","WARNING:paperscraper.load_dumps: No dump found for medrxiv. Skipping entry.\n","WARNING:paperscraper.load_dumps: No dumps found for either biorxiv or medrxiv. Consider using paperscraper.get_dumps.* to fetch the dumps.\n"]}],"source":["#@title Scrape paper informations\n","%%capture\n","!pip -q install pymed==0.8.9\n","!pip -q install paperscraper==0.2.10\n","!pip -q install arxivscraper\n","!pip -q install rich\n","\n","import arxivscraper, textwrap, json, torch\n","import numpy as np\n","import pandas as pd\n","from datetime import date, datetime, timedelta\n","from paperscraper.get_dumps import biorxiv, medrxiv, chemrxiv\n","from pymed import PubMed\n","from huggingface_hub import notebook_login\n","notebook_login()\n","import warnings\n","warnings.filterwarnings(\"ignore\")\n","\n","# date handling\n","def format_date(date, sep):\n","  assert len(sep) == 1\n","  return sep.join([date.strftime(f\"%{x}\") for x in \"Ymd\"])\n","\n","def format_dates(dates, sep):\n","  return [format_date(d, sep) for d in dates]\n","\n","def dates_between(start_date, end_date):\n","  \"\"\"\n","  Returns a list of all dates between two dates (inclusive).\n","  \"\"\"\n","  dates = []\n","  current_date = start_date\n","  while current_date <= end_date:\n","    dates.append(current_date)\n","    current_date += timedelta(days=1)\n","  return dates\n","\n","#get the last date when biblio was assessed\n","with open(\"/content/drive/MyDrive/WIP/Biblio/biblio_date.json\", \"r\") as f:\n","  date_dict = json.load(f)\n","date_start_str = date_dict['year']+ \"/\" + date_dict['month']+ \"/\" + date_dict['day']\n","date_start_datetime = datetime.strptime(date_start_str, \"%Y/%m/%d\")\n","\n","#rXiv dates\n","start_rxivs = format_date(date_start_datetime,  \"-\")\n","end_rxivs = format_date(date.today() - timedelta(days = 1), \"-\")\n","\n","#pubmed dates\n","dates = dates_between(date_start_datetime.date(), date.today() - timedelta(days = 1))\n","pubmed_days = [format_date(d, \"/\") for d in dates]\n","\n"]},{"cell_type":"code","source":["#@title Download paper dumps\n","#Download paper list from rXivs:\n","print(\"medRxiv:\")\n","medrxiv(begin_date=start_rxivs, end_date=end_rxivs, save_path=\"medrxiv.jsonl\")\n","print(\"bioRxiv:\")\n","biorxiv(begin_date=start_rxivs, end_date=end_rxivs, save_path=\"biorxiv.jsonl\")\n","print(\"chemRxiv:\")\n","chemrxiv(begin_date=start_rxivs, end_date=end_rxivs, save_path=\"chemrxiv.jsonl\")\n","\n","#scrape the arxiv q-bio papers\n","print(\"arXiv:\")\n","scraper = arxivscraper.Scraper(category='q-bio', date_from=start_rxivs, date_until=end_rxivs)\n","output = scraper.scrape()\n","cols = ('id', 'title', 'categories', 'abstract', 'doi', 'created', 'updated', 'authors')\n","df_arxiv = pd.DataFrame(output,columns=cols)"],"metadata":{"cellView":"form","id":"UxjXEaNCz1aZ","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1709887244168,"user_tz":-60,"elapsed":129791,"user":{"displayName":"Baptiste Roelens","userId":"10230221551692918814"}},"outputId":"4bbdbeca-a5df-4cfd-f3d7-2aa9c2d75114"},"execution_count":2,"outputs":[{"output_type":"stream","name":"stdout","text":["medRxiv:\n"]},{"output_type":"stream","name":"stderr","text":["107it [00:17,  6.29it/s]\n"]},{"output_type":"stream","name":"stdout","text":["bioRxiv:\n"]},{"output_type":"stream","name":"stderr","text":["382it [01:07,  5.70it/s]\n"]},{"output_type":"stream","name":"stdout","text":["chemRxiv:\n"]},{"output_type":"stream","name":"stderr","text":["58it [00:43,  1.35it/s]\n","100%|██████████| 60/60 [00:00<00:00, 4455.23it/s]\n"]},{"output_type":"stream","name":"stdout","text":["arXiv:\n","fetching up to  1000 records...\n","fetching is completed in 2.5 seconds.\n","Total number of records 54\n"]}]},{"cell_type":"code","source":["#@title Load classifier and tokenizer\n","from transformers import AutoModelForSequenceClassification, AutoTokenizer\n","import torch\n","\n","id2label = {0: \"Not Relevant\", 1: \"Potentially Interesting\"}\n","label2id = {\"Not Relevant\": 0, \"Potentially Interesting\": 1}\n","tokenizer = AutoTokenizer.from_pretrained(\"batroelens/PubMed_interests\")\n","model = AutoModelForSequenceClassification.from_pretrained(\"batroelens/PubMed_interests\", num_labels=2, id2label=id2label, label2id=label2id).to(\"cuda\")\n","\n","def preprocess_function(abstract):\n","    return tokenizer(abstract, truncation=True, max_length=512, return_tensors='pt').to(\"cuda\")\n","\n","data = {\"Title\": [], \"Abstract\": [], \"Relevance\": [], \"Journal\": [], \"URL\": []}\n","\n","for jsonfile in [\"medrxiv.jsonl\", \"biorxiv.jsonl\", \"chemrxiv.jsonl\"]:\n","  with open(jsonfile) as infile:\n","    for line in infile:\n","      l = json.loads(line)\n","      abstract = l[\"abstract\"].replace(\"\\n\", \" \")\n","      ab = preprocess_function(abstract)\n","      output = model(**ab)\n","      probabilities = torch.softmax(output.logits, dim=1)\n","      class_probabilities = probabilities[0].tolist()\n","      data[\"Title\"].append(l[\"title\"])\n","      data[\"Abstract\"].append(abstract)\n","      data[\"Relevance\"].append(class_probabilities[1])\n","      data[\"Journal\"].append(jsonfile.split(\".\")[0])\n","      data[\"URL\"].append(\"https://doi.org/\"+l[\"doi\"])\n","\n","data_pm = {\"Title\": [], \"Abstract\": [], \"Relevance\": [], \"Journal\": [], \"URL\": []}\n","pm_fail = False\n","\n","try:\n","  for pubmed_date in pubmed_days:\n","    pubmed = PubMed(tool=\"MyTool\", email=\"bla@bla.bla\")\n","    search_query = f\"{pubmed_date}[PDAT]\"\n","    results = pubmed.query(search_query, max_results=500)\n","    errors = []\n","    for i, article in enumerate(results):\n","      if article.abstract is None:\n","        continue\n","      ab = preprocess_function(article.abstract)\n","      output = model(**ab)\n","      probabilities = torch.softmax(output.logits, dim=1)\n","      class_probabilities = probabilities[0].tolist()\n","      data_pm[\"Title\"].append(article.title)\n","      data_pm[\"Abstract\"].append(article.abstract.replace(\"\\n\", \" \"))\n","      data_pm[\"Relevance\"].append(class_probabilities[1])\n","      try:\n","        data_pm[\"Journal\"].append(article.journal.strip().replace(\"\\n\", \" \"))\n","      except:\n","        data_pm[\"Journal\"].append(\"NA\")\n","      data_pm[\"URL\"].append(\"https://doi.org/\"+ article.doi.split(\"\\n\")[0])\n","except:\n","  pm_fail = True\n","  print(\"Pubmed analysis failed\")\n","\n","for index, row in df_arxiv.iterrows():\n","  ab = preprocess_function(df_arxiv.loc[index, 'abstract'])\n","  output = model(**ab)\n","  probabilities = torch.softmax(output.logits, dim=1)\n","  class_probabilities = probabilities[0].tolist()\n","  data[\"Title\"].append(df_arxiv.loc[index, 'title'])\n","  data[\"Abstract\"].append(df_arxiv.loc[index, 'abstract'].replace(\"\\n\", \" \"))\n","  data[\"Relevance\"].append(class_probabilities[1])\n","  data[\"Journal\"].append('arXiv')\n","  data[\"URL\"].append(\"https://doi.org/\"+ df_arxiv.loc[index, 'doi'])"],"metadata":{"cellView":"form","id":"yOaCLjNX1t3B","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1709887501850,"user_tz":-60,"elapsed":19844,"user":{"displayName":"Baptiste Roelens","userId":"10230221551692918814"}},"outputId":"cfd87807-1b45-4f1e-ef15-94c8e1191fce"},"execution_count":6,"outputs":[{"output_type":"stream","name":"stdout","text":["Pubmed analysis failed\n"]}]},{"cell_type":"code","source":["#@title Display relevant papers\n","from rich.console import Console\n","console = Console()\n","\n","if not pm_fail:\n","  for key, value in data.items():\n","    data[key] = value + data_pm[key]\n","\n","papers = pd.DataFrame.from_dict(data)\n","papers = papers.sort_values(\"Relevance\", ascending=False)\n","selected_papers = papers[papers[\"Relevance\"] >= 0.8]\n","\n","i=1\n","for _, row in selected_papers.iterrows():\n","  prob, title, journal, abstract, url =  row[\"Relevance\"], row[\"Title\"], row[\"Journal\"], row[\"Abstract\"], row[\"URL\"]\n","  console.print(f\"{i}- [bold]{title}[/bold] - [italic]{prob}[/italic] \\n [italic]{journal}[/italic] \\n [cyan]{url}[/cyan]\")\n","  i += 1"],"metadata":{"cellView":"form","id":"B2FZ8Nmq_l1H","colab":{"base_uri":"https://localhost:8080/","height":1000},"executionInfo":{"status":"ok","timestamp":1709887671752,"user_tz":-60,"elapsed":979,"user":{"displayName":"Baptiste Roelens","userId":"10230221551692918814"}},"outputId":"303e9f38-4618-4368-bba0-d8ee01ab828d"},"execution_count":7,"outputs":[{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m1\u001b[0m- \u001b[1mPRDM9 drives the location and rapid evolution of recombination hotspots in salmonids\u001b[0m - \u001b[1;3;36m0.9991962313652039\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.06.583651\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">1</span>- <span style=\"font-weight: bold\">PRDM9 drives the location and rapid evolution of recombination hotspots in salmonids</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9991962313652039</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.06.583651</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m2\u001b[0m- \u001b[1mRPA and Rad27 limit templated and inverted insertions at DNA breaks\u001b[0m - \u001b[1;3;36m0.9985656142234802\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.07.583931\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">2</span>- <span style=\"font-weight: bold\">RPA and Rad27 limit templated and inverted insertions at DNA breaks</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9985656142234802</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.07.583931</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m3\u001b[0m- \u001b[1mPrevalent fast evolution of genes involved in heterochromatin functions\u001b[0m - \u001b[1;3;36m0.9985034465789795\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.03.583199\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">3</span>- <span style=\"font-weight: bold\">Prevalent fast evolution of genes involved in heterochromatin functions</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9985034465789795</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.03.583199</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m4\u001b[0m- \u001b[1mEpicTope: narrating protein sequence features to identify non-disruptive epitope tagging sites\u001b[0m - \n","\u001b[1;3;36m0.998357355594635\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.03.583232\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">4</span>- <span style=\"font-weight: bold\">EpicTope: narrating protein sequence features to identify non-disruptive epitope tagging sites</span> - \n","<span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.998357355594635</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.03.583232</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m5\u001b[0m- \u001b[1mEDC-\u001b[0m\u001b[1;36m3\u001b[0m\u001b[1m and EDC-\u001b[0m\u001b[1;36m4\u001b[0m\u001b[1m Regulate Embryonic mRNA Clearance and Biomolecular Condensate Specialization\u001b[0m - \n","\u001b[1;3;36m0.9981433153152466\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.04.583404\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">5</span>- <span style=\"font-weight: bold\">EDC-</span><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">3</span><span style=\"font-weight: bold\"> and EDC-</span><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">4</span><span style=\"font-weight: bold\"> Regulate Embryonic mRNA Clearance and Biomolecular Condensate Specialization</span> - \n","<span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9981433153152466</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.04.583404</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m6\u001b[0m- \u001b[1mBiochemical properties of chromatin domains define genome compartmentalization\u001b[0m - \u001b[1;3;36m0.9977508187294006\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.05.583467\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">6</span>- <span style=\"font-weight: bold\">Biochemical properties of chromatin domains define genome compartmentalization</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9977508187294006</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.05.583467</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m7\u001b[0m- \u001b[1mAn Aurora kinase A-BOD1L1-PP2A B56 Axis promotes chromosome segregation fidelity\u001b[0m - \u001b[1;3;36m0.9972520470619202\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2023.08.06.552174\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">7</span>- <span style=\"font-weight: bold\">An Aurora kinase A-BOD1L1-PP2A B56 Axis promotes chromosome segregation fidelity</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9972520470619202</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2023.08.06.552174</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m8\u001b[0m- \u001b[1mMitotic waves in frog egg extracts: Transition from phase waves to trigger waves\u001b[0m - \u001b[1;3;36m0.9960190653800964\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.01.18.576267\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">8</span>- <span style=\"font-weight: bold\">Mitotic waves in frog egg extracts: Transition from phase waves to trigger waves</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9960190653800964</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.01.18.576267</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m9\u001b[0m- \u001b[1mdiffusion on language model embeddings for protein sequence generation\u001b[0m - \u001b[1;3;36m0.9959873557090759\u001b[0m \n"," \u001b[3marXiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">9</span>- <span style=\"font-weight: bold\">diffusion on language model embeddings for protein sequence generation</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9959873557090759</span> \n"," <span style=\"font-style: italic\">arXiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m10\u001b[0m- \u001b[1mNucleolar stress induced the formation of a nucleolar stress body via the NOSR-\u001b[0m\u001b[1;36m1\u001b[0m\u001b[1m/NUMR-\u001b[0m\u001b[1;36m1\u001b[0m\u001b[1m axis in Caenorhabditis \u001b[0m\n","\u001b[1melegans\u001b[0m - \u001b[1;3;36m0.9958925247192383\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.05.583293\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">10</span>- <span style=\"font-weight: bold\">Nucleolar stress induced the formation of a nucleolar stress body via the NOSR-</span><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">1</span><span style=\"font-weight: bold\">/NUMR-</span><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">1</span><span style=\"font-weight: bold\"> axis in Caenorhabditis </span>\n","<span style=\"font-weight: bold\">elegans</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9958925247192383</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.05.583293</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m11\u001b[0m- \u001b[1mA male-essential microRNA is key for avian sex chromosome dosage compensation\u001b[0m - \u001b[1;3;36m0.9955875873565674\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.06.581755\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">11</span>- <span style=\"font-weight: bold\">A male-essential microRNA is key for avian sex chromosome dosage compensation</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9955875873565674</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.06.581755</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m12\u001b[0m- \u001b[1mNon-viral Intron Knockins Enable Simplified and Flexible Targeting of Endogenous Genes\u001b[0m - \u001b[1;3;36m0.9955548644065857\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.05.582227\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">12</span>- <span style=\"font-weight: bold\">Non-viral Intron Knockins Enable Simplified and Flexible Targeting of Endogenous Genes</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9955548644065857</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.05.582227</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m13\u001b[0m- \u001b[1mabdiffuser: full-atom generation of in vitro functioning antibodies\u001b[0m - \u001b[1;3;36m0.9948210716247559\u001b[0m \n"," \u001b[3marXiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">13</span>- <span style=\"font-weight: bold\">abdiffuser: full-atom generation of in vitro functioning antibodies</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9948210716247559</span> \n"," <span style=\"font-style: italic\">arXiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m14\u001b[0m- \u001b[1mComputational design of soluble functional analogues of integral membrane proteins\u001b[0m - \u001b[1;3;36m0.99444580078125\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2023.05.09.540044\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">14</span>- <span style=\"font-weight: bold\">Computational design of soluble functional analogues of integral membrane proteins</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.99444580078125</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2023.05.09.540044</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m15\u001b[0m- \u001b[1mcommunity recommendations on cryoem data archiving and validation\u001b[0m - \u001b[1;3;36m0.9943312406539917\u001b[0m \n"," \u001b[3marXiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1107/s2052252524001246\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">15</span>- <span style=\"font-weight: bold\">community recommendations on cryoem data archiving and validation</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9943312406539917</span> \n"," <span style=\"font-style: italic\">arXiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1107/s2052252524001246</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m16\u001b[0m- \u001b[1mCis inhibition of co-expressed DIPs and Dprs shapes neural development\u001b[0m - \u001b[1;3;36m0.9943140745162964\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.04.583391\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">16</span>- <span style=\"font-weight: bold\">Cis inhibition of co-expressed DIPs and Dprs shapes neural development</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9943140745162964</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.04.583391</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m17\u001b[0m- \u001b[1mComparative analysis of retroviral Gag-host cell interactions: focus on the nuclear interactome\u001b[0m - \n","\u001b[1;3;36m0.9939998388290405\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.01.18.575255\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">17</span>- <span style=\"font-weight: bold\">Comparative analysis of retroviral Gag-host cell interactions: focus on the nuclear interactome</span> - \n","<span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9939998388290405</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.01.18.575255</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m18\u001b[0m- \u001b[1mA Structured Illumination Microscopy Framework with Spatial-Domain Noise Propagation.\u001b[0m - \u001b[1;3;36m0.9929805994033813\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2023.12.07.570701\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">18</span>- <span style=\"font-weight: bold\">A Structured Illumination Microscopy Framework with Spatial-Domain Noise Propagation.</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9929805994033813</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2023.12.07.570701</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m19\u001b[0m- \u001b[1mDefining the EM-signature of successful cell-transfection\u001b[0m - \u001b[1;3;36m0.9929465651512146\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.07.583927\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">19</span>- <span style=\"font-weight: bold\">Defining the EM-signature of successful cell-transfection</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9929465651512146</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.07.583927</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m20\u001b[0m- \u001b[1mAccurate prediction of site- and amino-acid substitution rates with a mutation-selection model\u001b[0m - \n","\u001b[1;3;36m0.9928473830223083\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.02.583099\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">20</span>- <span style=\"font-weight: bold\">Accurate prediction of site- and amino-acid substitution rates with a mutation-selection model</span> - \n","<span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9928473830223083</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.02.583099</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m21\u001b[0m- \u001b[1mSubtomogram averaging of helical filaments in cryo-electron tomography\u001b[0m - \u001b[1;3;36m0.9921537041664124\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.04.583254\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">21</span>- <span style=\"font-weight: bold\">Subtomogram averaging of helical filaments in cryo-electron tomography</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9921537041664124</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.04.583254</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m22\u001b[0m- \u001b[1mIdentification of ESCRT-III like protein sequences in yeast\u001b[0m - \u001b[1;3;36m0.9921481013298035\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.07.583924\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">22</span>- <span style=\"font-weight: bold\">Identification of ESCRT-III like protein sequences in yeast</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9921481013298035</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.07.583924</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m23\u001b[0m- \u001b[1mexplicitly explainable ai solution to the ai black box problem\u001b[0m - \u001b[1;3;36m0.9919079542160034\u001b[0m \n"," \u001b[3marXiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">23</span>- <span style=\"font-weight: bold\">explicitly explainable ai solution to the ai black box problem</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9919079542160034</span> \n"," <span style=\"font-style: italic\">arXiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m24\u001b[0m- \u001b[1mProteogenomic analysis reveals adaptive strategies to alleviate the consequences of aneuploidy in cancer\u001b[0m - \n","\u001b[1;3;36m0.9918805956840515\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.05.583460\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">24</span>- <span style=\"font-weight: bold\">Proteogenomic analysis reveals adaptive strategies to alleviate the consequences of aneuploidy in cancer</span> - \n","<span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9918805956840515</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.05.583460</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m25\u001b[0m- \u001b[1mVisualization of endogenous G proteins on endosomes and other organelles\u001b[0m - \u001b[1;3;36m0.9885144829750061\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.05.583500\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">25</span>- <span style=\"font-weight: bold\">Visualization of endogenous G proteins on endosomes and other organelles</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9885144829750061</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.05.583500</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m26\u001b[0m- \u001b[1mInefficient Endocytosis Limits Heterotrimeric G Protein Abundance on Endosomes\u001b[0m - \u001b[1;3;36m0.9885144829750061\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.05.583500\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">26</span>- <span style=\"font-weight: bold\">Inefficient Endocytosis Limits Heterotrimeric G Protein Abundance on Endosomes</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9885144829750061</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.05.583500</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m27\u001b[0m- \u001b[1mIMPROVING ALPHAFOLD2 PERFORMANCE WITH A GLOBAL METAGENOMIC & BIOLOGICAL DATA SUPPLY CHAIN\u001b[0m - \u001b[1;3;36m0.988108217716217\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.06.583325\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">27</span>- <span style=\"font-weight: bold\">IMPROVING ALPHAFOLD2 PERFORMANCE WITH A GLOBAL METAGENOMIC &amp; BIOLOGICAL DATA SUPPLY CHAIN</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.988108217716217</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.06.583325</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m28\u001b[0m- \u001b[1mCritical factors for precise and efficient RNA cleavage by RNase Y in Staphylococcus aureus\u001b[0m - \n","\u001b[1;3;36m0.9874789714813232\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2023.08.22.554078\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">28</span>- <span style=\"font-weight: bold\">Critical factors for precise and efficient RNA cleavage by RNase Y in Staphylococcus aureus</span> - \n","<span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9874789714813232</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2023.08.22.554078</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m29\u001b[0m- \u001b[1mProtein Embeddings Predict Binding Residues in Disordered Regions\u001b[0m - \u001b[1;3;36m0.9855175614356995\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.05.583540\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">29</span>- <span style=\"font-weight: bold\">Protein Embeddings Predict Binding Residues in Disordered Regions</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9855175614356995</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.05.583540</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m30\u001b[0m- \u001b[1mon the use of associative memory in hopfield networks designed to solve   propositional satisfiability problems\u001b[0m\n","- \u001b[1;3;36m0.9828460812568665\u001b[0m \n"," \u001b[3marXiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1109/ssci52147.2023.10371918\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">30</span>- <span style=\"font-weight: bold\">on the use of associative memory in hopfield networks designed to solve   propositional satisfiability problems</span>\n","- <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9828460812568665</span> \n"," <span style=\"font-style: italic\">arXiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1109/ssci52147.2023.10371918</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m31\u001b[0m- \u001b[1mNuclear RNAi Modulates Influenza A Virus Infectivity By Downregulating Type-I Interferon Response\u001b[0m - \n","\u001b[1;3;36m0.9827097654342651\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.07.583365\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">31</span>- <span style=\"font-weight: bold\">Nuclear RNAi Modulates Influenza A Virus Infectivity By Downregulating Type-I Interferon Response</span> - \n","<span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9827097654342651</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.07.583365</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m32\u001b[0m- \u001b[1mTransformer-Based Deep Learning Model with Latent Space Regularization for CRISPR-Cas Protein Sequence \u001b[0m\n","\u001b[1mClassification\u001b[0m - \u001b[1;3;36m0.9809169173240662\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.02.583136\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">32</span>- <span style=\"font-weight: bold\">Transformer-Based Deep Learning Model with Latent Space Regularization for CRISPR-Cas Protein Sequence </span>\n","<span style=\"font-weight: bold\">Classification</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9809169173240662</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.02.583136</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m33\u001b[0m- \u001b[1mEfficient acceleration of the convergence of the Minimum Free Energy Path via path-planning generated initial \u001b[0m\n","\u001b[1mguess\u001b[0m - \u001b[1;3;36m0.9803224205970764\u001b[0m \n"," \u001b[3mchemrxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.26434/chemrxiv-2024-58tt1\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">33</span>- <span style=\"font-weight: bold\">Efficient acceleration of the convergence of the Minimum Free Energy Path via path-planning generated initial </span>\n","<span style=\"font-weight: bold\">guess</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9803224205970764</span> \n"," <span style=\"font-style: italic\">chemrxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.26434/chemrxiv-2024-58tt1</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m34\u001b[0m- \u001b[1mThe First CACHE Challenge – Testing Diverse Virtual Screening Scoring Methods to Identify Potential LRRK2 \u001b[0m\n","\u001b[1mBinders\u001b[0m - \u001b[1;3;36m0.9798802733421326\u001b[0m \n"," \u001b[3mchemrxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.26434/chemrxiv-2022-ncqsj-v3\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">34</span>- <span style=\"font-weight: bold\">The First CACHE Challenge – Testing Diverse Virtual Screening Scoring Methods to Identify Potential LRRK2 </span>\n","<span style=\"font-weight: bold\">Binders</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9798802733421326</span> \n"," <span style=\"font-style: italic\">chemrxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.26434/chemrxiv-2022-ncqsj-v3</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m35\u001b[0m- \u001b[1mDUAL: deep unsupervised simultaneous simulation and denoising for cryo-electron tomography\u001b[0m - \u001b[1;3;36m0.979290783405304\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.02.583135\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">35</span>- <span style=\"font-weight: bold\">DUAL: deep unsupervised simultaneous simulation and denoising for cryo-electron tomography</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.979290783405304</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.02.583135</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m36\u001b[0m- \u001b[1mcrispr: ensemble model\u001b[0m - \u001b[1;3;36m0.9773257970809937\u001b[0m \n"," \u001b[3marXiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">36</span>- <span style=\"font-weight: bold\">crispr: ensemble model</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9773257970809937</span> \n"," <span style=\"font-style: italic\">arXiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m37\u001b[0m- \u001b[1mHuman immunodeficiency virus-\u001b[0m\u001b[1;36m1\u001b[0m\u001b[1m induces and targets host genomic R-loops for viral genome integration\u001b[0m - \n","\u001b[1;3;36m0.9763906002044678\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.06.583715\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">37</span>- <span style=\"font-weight: bold\">Human immunodeficiency virus-</span><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">1</span><span style=\"font-weight: bold\"> induces and targets host genomic R-loops for viral genome integration</span> - \n","<span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9763906002044678</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.06.583715</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m38\u001b[0m- \u001b[1mOverlapping binding sites underlie TF genomic occupancy\u001b[0m - \u001b[1;3;36m0.9747263193130493\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.05.583629\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">38</span>- <span style=\"font-weight: bold\">Overlapping binding sites underlie TF genomic occupancy</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9747263193130493</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.05.583629</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m39\u001b[0m- \u001b[1mLink between lipid remodeling and ESCRT-III function in multivesicular body formation\u001b[0m - \u001b[1;3;36m0.9735715389251709\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2023.10.10.561655\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">39</span>- <span style=\"font-weight: bold\">Link between lipid remodeling and ESCRT-III function in multivesicular body formation</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9735715389251709</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2023.10.10.561655</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m40\u001b[0m- \u001b[1mSequence modeling and design from molecular to genome scale with Evo\u001b[0m - \u001b[1;3;36m0.9733957648277283\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.02.27.582234\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">40</span>- <span style=\"font-weight: bold\">Sequence modeling and design from molecular to genome scale with Evo</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9733957648277283</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.02.27.582234</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m41\u001b[0m- \u001b[1mDeltaGzip: Computing Biopolymer-Ligand Binding Affinity via Kolmogorov Complexity and Lossless Compression\u001b[0m - \n","\u001b[1;3;36m0.971498429775238\u001b[0m \n"," \u001b[3mchemrxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.26434/chemrxiv-2024-cq702\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">41</span>- <span style=\"font-weight: bold\">DeltaGzip: Computing Biopolymer-Ligand Binding Affinity via Kolmogorov Complexity and Lossless Compression</span> - \n","<span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.971498429775238</span> \n"," <span style=\"font-style: italic\">chemrxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.26434/chemrxiv-2024-cq702</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m42\u001b[0m- \u001b[1mThe mutational landscape of Bacillus subtilis conditional hypermutators shows how proofreading skews DNA \u001b[0m\n","\u001b[1mpolymerase error rates\u001b[0m - \u001b[1;3;36m0.9709904789924622\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2023.12.29.573609\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">42</span>- <span style=\"font-weight: bold\">The mutational landscape of Bacillus subtilis conditional hypermutators shows how proofreading skews DNA </span>\n","<span style=\"font-weight: bold\">polymerase error rates</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9709904789924622</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2023.12.29.573609</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m43\u001b[0m- \u001b[1mHaplotype-resolved karyotype construction from Hi-C data using refLinker\u001b[0m - \u001b[1;3;36m0.9681727886199951\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.02.583108\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">43</span>- <span style=\"font-weight: bold\">Haplotype-resolved karyotype construction from Hi-C data using refLinker</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9681727886199951</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.02.583108</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m44\u001b[0m- \u001b[1mComputation or Weight Adaptation? Rethinking the Role of Plasticity in Learning\u001b[0m - \u001b[1;3;36m0.9662463665008545\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.07.583890\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">44</span>- <span style=\"font-weight: bold\">Computation or Weight Adaptation? Rethinking the Role of Plasticity in Learning</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9662463665008545</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.07.583890</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m45\u001b[0m- \u001b[1mMulti-Scale Protein Language Model for Unified Molecular Modeling\u001b[0m - \u001b[1;3;36m0.9643686413764954\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.04.583284\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">45</span>- <span style=\"font-weight: bold\">Multi-Scale Protein Language Model for Unified Molecular Modeling</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9643686413764954</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.04.583284</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m46\u001b[0m- \u001b[1mDual-color MINFLUX: Kinesin-\u001b[0m\u001b[1;36m1\u001b[0m\u001b[1m takes Chasse-Inchworm steps\u001b[0m - \u001b[1;3;36m0.9641490578651428\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.05.583551\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">46</span>- <span style=\"font-weight: bold\">Dual-color MINFLUX: Kinesin-</span><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">1</span><span style=\"font-weight: bold\"> takes Chasse-Inchworm steps</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9641490578651428</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.05.583551</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m47\u001b[0m- \u001b[1mGUANinE v1.\u001b[0m\u001b[1;36m0\u001b[0m\u001b[1m: Benchmark Datasets for Genomic AISequence-to-Function Models\u001b[0m - \u001b[1;3;36m0.9639616012573242\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2023.10.12.562113\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">47</span>- <span style=\"font-weight: bold\">GUANinE v1.</span><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">0</span><span style=\"font-weight: bold\">: Benchmark Datasets for Genomic AISequence-to-Function Models</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9639616012573242</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2023.10.12.562113</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m48\u001b[0m- \u001b[1mspanseq: similarity-based sequence data splitting method for improved   development and assessment of deep \u001b[0m\n","\u001b[1mlearning projects\u001b[0m - \u001b[1;3;36m0.9625247120857239\u001b[0m \n"," \u001b[3marXiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">48</span>- <span style=\"font-weight: bold\">spanseq: similarity-based sequence data splitting method for improved   development and assessment of deep </span>\n","<span style=\"font-weight: bold\">learning projects</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9625247120857239</span> \n"," <span style=\"font-style: italic\">arXiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m49\u001b[0m- \u001b[1mAMP-Diffusion: Integrating Latent Diffusion with Protein Language Models for Antimicrobial Peptide Generation\u001b[0m -\n","\u001b[1;3;36m0.9621661305427551\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.03.583201\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">49</span>- <span style=\"font-weight: bold\">AMP-Diffusion: Integrating Latent Diffusion with Protein Language Models for Antimicrobial Peptide Generation</span> -\n","<span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9621661305427551</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.03.583201</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m50\u001b[0m- \u001b[1mfixation times on directed graphs\u001b[0m - \u001b[1;3;36m0.9579368233680725\u001b[0m \n"," \u001b[3marXiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">50</span>- <span style=\"font-weight: bold\">fixation times on directed graphs</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9579368233680725</span> \n"," <span style=\"font-style: italic\">arXiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m51\u001b[0m- \u001b[1mconsciousness qua mortal computation\u001b[0m - \u001b[1;3;36m0.9578775763511658\u001b[0m \n"," \u001b[3marXiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">51</span>- <span style=\"font-weight: bold\">consciousness qua mortal computation</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9578775763511658</span> \n"," <span style=\"font-style: italic\">arXiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m52\u001b[0m- \u001b[1mSpatiotemporal regulation of Pseudorabies Virus Thymidine kinase UL23 by post-translational modification\u001b[0m - \n","\u001b[1;3;36m0.9554551839828491\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.06.583799\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">52</span>- <span style=\"font-weight: bold\">Spatiotemporal regulation of Pseudorabies Virus Thymidine kinase UL23 by post-translational modification</span> - \n","<span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9554551839828491</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.06.583799</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m53\u001b[0m- \u001b[1mProximity Biosensor Assay for PROTAC Ternary Complex Analysis\u001b[0m - \u001b[1;3;36m0.955355167388916\u001b[0m \n"," \u001b[3mchemrxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.26434/chemrxiv-2024-8w4zb\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">53</span>- <span style=\"font-weight: bold\">Proximity Biosensor Assay for PROTAC Ternary Complex Analysis</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.955355167388916</span> \n"," <span style=\"font-style: italic\">chemrxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.26434/chemrxiv-2024-8w4zb</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m54\u001b[0m- \u001b[1mDecoding Stability and Epistasis in Human Myoglobin by Deep Mutational Scanning and Codon-level Machine \u001b[0m\n","\u001b[1mLearning\u001b[0m - \u001b[1;3;36m0.9547705054283142\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.02.24.581358\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">54</span>- <span style=\"font-weight: bold\">Decoding Stability and Epistasis in Human Myoglobin by Deep Mutational Scanning and Codon-level Machine </span>\n","<span style=\"font-weight: bold\">Learning</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9547705054283142</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.02.24.581358</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m55\u001b[0m- \u001b[1mA Transformer Based Machine Learning of Molecular Grammar Inherent in Proteins Prone to Liquid Liquid Phase \u001b[0m\n","\u001b[1mSeparation\u001b[0m - \u001b[1;3;36m0.9435614943504333\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.02.583105\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">55</span>- <span style=\"font-weight: bold\">A Transformer Based Machine Learning of Molecular Grammar Inherent in Proteins Prone to Liquid Liquid Phase </span>\n","<span style=\"font-weight: bold\">Separation</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9435614943504333</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.02.583105</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m56\u001b[0m- \u001b[1mKnotted artifacts in predicted 3D RNA structures\u001b[0m - \u001b[1;3;36m0.9427907466888428\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.04.583268\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">56</span>- <span style=\"font-weight: bold\">Knotted artifacts in predicted 3D RNA structures</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9427907466888428</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.04.583268</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m57\u001b[0m- \u001b[1mKnotted artifacts in predicted 3D RNA structures\u001b[0m - \u001b[1;3;36m0.9427907466888428\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.04.583268\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">57</span>- <span style=\"font-weight: bold\">Knotted artifacts in predicted 3D RNA structures</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9427907466888428</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.04.583268</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m58\u001b[0m- \u001b[1mMapping the mutational landscape of a full viral proteome reveals distinct profiles of mutation tolerability\u001b[0m - \n","\u001b[1;3;36m0.9410009980201721\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.07.583990\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">58</span>- <span style=\"font-weight: bold\">Mapping the mutational landscape of a full viral proteome reveals distinct profiles of mutation tolerability</span> - \n","<span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9410009980201721</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.07.583990</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m59\u001b[0m- \u001b[1mAltered X-chromosome inactivation predisposes to autoimmunity\u001b[0m - \u001b[1;3;36m0.9388387799263\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2023.04.20.537662\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">59</span>- <span style=\"font-weight: bold\">Altered X-chromosome inactivation predisposes to autoimmunity</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9388387799263</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2023.04.20.537662</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m60\u001b[0m- \u001b[1mearly quantum computing applications on the path towards precision   medicine\u001b[0m - \u001b[1;3;36m0.9363693594932556\u001b[0m \n"," \u001b[3marXiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">60</span>- <span style=\"font-weight: bold\">early quantum computing applications on the path towards precision   medicine</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9363693594932556</span> \n"," <span style=\"font-style: italic\">arXiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m61\u001b[0m- \u001b[1munstable cores are the source of instability in chemical reaction   networks\u001b[0m - \u001b[1;3;36m0.9262908101081848\u001b[0m \n"," \u001b[3marXiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1098/rspa.2023.0694\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">61</span>- <span style=\"font-weight: bold\">unstable cores are the source of instability in chemical reaction   networks</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9262908101081848</span> \n"," <span style=\"font-style: italic\">arXiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1098/rspa.2023.0694</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m62\u001b[0m- \u001b[1mA Foundation Model for Cell Segmentation\u001b[0m - \u001b[1;3;36m0.9219130873680115\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2023.11.17.567630\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">62</span>- <span style=\"font-weight: bold\">A Foundation Model for Cell Segmentation</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9219130873680115</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2023.11.17.567630</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m63\u001b[0m- \u001b[1mImproved selection of canonical proteins for reference proteomes\u001b[0m - \u001b[1;3;36m0.9191962480545044\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.04.583387\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">63</span>- <span style=\"font-weight: bold\">Improved selection of canonical proteins for reference proteomes</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9191962480545044</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.04.583387</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m64\u001b[0m- \u001b[1mRegulation of virion production by the ORF8 signal peptide across SARS-CoV-\u001b[0m\u001b[1;36m2\u001b[0m\u001b[1m variants\u001b[0m - \u001b[1;3;36m0.9167857766151428\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.05.583578\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">64</span>- <span style=\"font-weight: bold\">Regulation of virion production by the ORF8 signal peptide across SARS-CoV-</span><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">2</span><span style=\"font-weight: bold\"> variants</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9167857766151428</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.05.583578</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m65\u001b[0m- \u001b[1mComparative analysis of Wolbachia maternal transmission and localization in host ovaries\u001b[0m - \u001b[1;3;36m0.9082450866699219\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.03.583170\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">65</span>- <span style=\"font-weight: bold\">Comparative analysis of Wolbachia maternal transmission and localization in host ovaries</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9082450866699219</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.03.583170</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m66\u001b[0m- \u001b[1mProbing recombinant AAV capsid integrity and genome release under thermal stress by single-molecule \u001b[0m\n","\u001b[1minterferometric scattering microscopy\u001b[0m - \u001b[1;3;36m0.9056702852249146\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.07.583968\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">66</span>- <span style=\"font-weight: bold\">Probing recombinant AAV capsid integrity and genome release under thermal stress by single-molecule </span>\n","<span style=\"font-weight: bold\">interferometric scattering microscopy</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9056702852249146</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.07.583968</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m67\u001b[0m- \u001b[1mlarge language models surpass human experts in predicting neuroscience   results\u001b[0m - \u001b[1;3;36m0.9040111899375916\u001b[0m \n"," \u001b[3marXiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">67</span>- <span style=\"font-weight: bold\">large language models surpass human experts in predicting neuroscience   results</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.9040111899375916</span> \n"," <span style=\"font-style: italic\">arXiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m68\u001b[0m- \u001b[1mCrimean-Congo Hemorrhagic Fever Survivors Elicit Protective Non-Neutralizing Antibodies that Target \u001b[0m\u001b[1;36m11\u001b[0m\u001b[1m \u001b[0m\n","\u001b[1mOverlapping Regions on Viral Glycoprotein GP38\u001b[0m - \u001b[1;3;36m0.901760995388031\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.02.583110\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">68</span>- <span style=\"font-weight: bold\">Crimean-Congo Hemorrhagic Fever Survivors Elicit Protective Non-Neutralizing Antibodies that Target </span><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">11</span><span style=\"font-weight: bold\"> </span>\n","<span style=\"font-weight: bold\">Overlapping Regions on Viral Glycoprotein GP38</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.901760995388031</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.02.583110</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m69\u001b[0m- \u001b[1mDevelopment of a Bipyrimidineamide based α-Helix Mimetic Lead Compound for efficient Targeting of MDM2 in \u001b[0m\n","\u001b[1mTriple-Negative Breast Cancer\u001b[0m - \u001b[1;3;36m0.8983370661735535\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.02.582899\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">69</span>- <span style=\"font-weight: bold\">Development of a Bipyrimidineamide based α-Helix Mimetic Lead Compound for efficient Targeting of MDM2 in </span>\n","<span style=\"font-weight: bold\">Triple-Negative Breast Cancer</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.8983370661735535</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.02.582899</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m70\u001b[0m- \u001b[1mSubnanometer structure of medusavirus capsid during maturation using cryo-electron microscopy\u001b[0m - \n","\u001b[1;3;36m0.8816971778869629\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.07.583866\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">70</span>- <span style=\"font-weight: bold\">Subnanometer structure of medusavirus capsid during maturation using cryo-electron microscopy</span> - \n","<span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.8816971778869629</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.07.583866</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m71\u001b[0m- \u001b[1mSubnanometer structure of medusavirus capsid during maturation using cryo-electron microscopy\u001b[0m - \n","\u001b[1;3;36m0.8816971778869629\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.07.583866\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">71</span>- <span style=\"font-weight: bold\">Subnanometer structure of medusavirus capsid during maturation using cryo-electron microscopy</span> - \n","<span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.8816971778869629</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.07.583866</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m72\u001b[0m- \u001b[1ma genome-scale deep learning model to predict gene expression changes of   genetic perturbations from multiplex\u001b[0m\n","\u001b[1mbiological networks\u001b[0m - \u001b[1;3;36m0.8792483806610107\u001b[0m \n"," \u001b[3marXiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">72</span>- <span style=\"font-weight: bold\">a genome-scale deep learning model to predict gene expression changes of   genetic perturbations from multiplex</span>\n","<span style=\"font-weight: bold\">biological networks</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.8792483806610107</span> \n"," <span style=\"font-style: italic\">arXiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m73\u001b[0m- \u001b[1mENT3C: an entropy-based similarity measure for Hi-C and micro-C derived contact matrices\u001b[0m - \u001b[1;3;36m0.8677976131439209\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.01.30.577923\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">73</span>- <span style=\"font-weight: bold\">ENT3C: an entropy-based similarity measure for Hi-C and micro-C derived contact matrices</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.8677976131439209</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.01.30.577923</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m74\u001b[0m- \u001b[1mCryo-EM structure of the tetra-phosphorylated R-domain in Ycf1 reveals key interactions for transport \u001b[0m\n","\u001b[1mregulation\u001b[0m - \u001b[1;3;36m0.8623061180114746\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.06.583773\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">74</span>- <span style=\"font-weight: bold\">Cryo-EM structure of the tetra-phosphorylated R-domain in Ycf1 reveals key interactions for transport </span>\n","<span style=\"font-weight: bold\">regulation</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.8623061180114746</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.06.583773</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m75\u001b[0m- \u001b[1mcaduceus: bi-directional equivariant long-range dna sequence modeling\u001b[0m - \u001b[1;3;36m0.8584968447685242\u001b[0m \n"," \u001b[3marXiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">75</span>- <span style=\"font-weight: bold\">caduceus: bi-directional equivariant long-range dna sequence modeling</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.8584968447685242</span> \n"," <span style=\"font-style: italic\">arXiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m76\u001b[0m- \u001b[1mInducible deletion of Ezh2 in CD4+ T cells inhibits kidney T cell infiltration and prevents interstitial \u001b[0m\n","\u001b[1mnephritis in MRL/lpr lupus-prone mice\u001b[0m - \u001b[1;3;36m0.8253605365753174\u001b[0m \n"," \u001b[3mbiorxiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/10.1101/2024.03.04.583401\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">76</span>- <span style=\"font-weight: bold\">Inducible deletion of Ezh2 in CD4+ T cells inhibits kidney T cell infiltration and prevents interstitial </span>\n","<span style=\"font-weight: bold\">nephritis in MRL/lpr lupus-prone mice</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.8253605365753174</span> \n"," <span style=\"font-style: italic\">biorxiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/10.1101/2024.03.04.583401</span>\n","</pre>\n"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["\u001b[1;36m77\u001b[0m- \u001b[1ma novel method for clustering cellular data to improve classification\u001b[0m - \u001b[1;3;36m0.8161166310310364\u001b[0m \n"," \u001b[3marXiv\u001b[0m \n"," \u001b[4;36mhttps://doi.org/\u001b[0m\n"],"text/html":["<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">77</span>- <span style=\"font-weight: bold\">a novel method for clustering cellular data to improve classification</span> - <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold; font-style: italic\">0.8161166310310364</span> \n"," <span style=\"font-style: italic\">arXiv</span> \n"," <span style=\"color: #008080; text-decoration-color: #008080; text-decoration: underline\">https://doi.org/</span>\n","</pre>\n"]},"metadata":{}}]},{"cell_type":"code","source":["#@title Format for adding to TSV for subsequent days.\n","#@markdown Manually set the value at the start of each row to one to indicate a positive example.\n","for _, row in selected_papers.iterrows():\n","  prob, title, journal, abstract =  row[\"Relevance\"], row[\"Title\"], row[\"Journal\"], row[\"Abstract\"]\n","  print(f\"{int(round(prob))} \\t{title} \\t{abstract}\")"],"metadata":{"cellView":"form","id":"FHYn78tkvA45","executionInfo":{"status":"aborted","timestamp":1709887280892,"user_tz":-60,"elapsed":5,"user":{"displayName":"Baptiste Roelens","userId":"10230221551692918814"}}},"execution_count":null,"outputs":[]},{"cell_type":"code","source":["#@title update stored date in json and disconnect runtime\n","from datetime import date, datetime, timedelta\n","\n","today = date.today()\n","new_date = {'year': str(today.strftime(\"%Y\")), 'month': str(today.strftime(\"%m\")), 'day': str(today.strftime(\"%d\"))}\n","\n","\n","json_date = '/content/drive/MyDrive/WIP/Biblio/biblio_date.json'\n","with open(json_date, \"w\") as f:\n","  json.dump(new_date, f, indent=4)\n","\n","\n","from google.colab import runtime\n","runtime.unassign()"],"metadata":{"cellView":"form","id":"wOfApDJrB4_W","executionInfo":{"status":"aborted","timestamp":1709887280893,"user_tz":-60,"elapsed":4,"user":{"displayName":"Baptiste Roelens","userId":"10230221551692918814"}}},"execution_count":null,"outputs":[]}]}